### ISSUE TITLE
Connection Reset By Peer

### ISSUE THREAD

**Issue by sovitacharya** [jackc/pgx#1140]
Hello,
While Using the CRDB and querying the database using Golang For required operations I am encountering the issue of "Connection Reset by Peer ", The following issue I have started encountering just after using a TCP Load Balancer For The 3 nodes of my crdb.
Also the issue can be simply return that the connection which I am acquiring from the pool is getting disconnected/ reset by the peers and collectively there is no issue I encounter with the connection pool made to the CRDB and still my application made works fine even if i called for a db connection(pool) for a single time only.

Any sort of reference or solution provided will be highly appreciated.
Thanks in Advance

**Comment by jackc** [comment]
It's a little hard to follow exactly whats happening from that description, but since you mention using a load balancer you may want to try using the simple protocol. That is sometimes needed when PgBouncer is in the mix.
